### ISSUE TITLE
Disabling default generated breadcrumbs by passing breadcrumb: null raises an error

### ISSUE THREAD

**Issue by rwojnarowski** [icd2k3/react-router-breadcrumbs-hoc#109]
According to docs (option 2) https://github.com/icd2k3/react-router-breadcrumbs-hoc#disabling-default-generated-breadcrumbs im able to disable breadcrumb by passing somthing like this:
`{ path: '/a/b', breadcrumb: null }`. And it's works but typescript type checker raises an error/warning:

```
Argument of type '({ path: string; breadcrumb: string; } | { path: string; breadcrumb: null; })[]' is not assignable to parameter of type 'BreadcrumbsRoute[]'.
  Type '{ path: string; breadcrumb: string; } | { path: string; breadcrumb: null; }' is not assignable to type 'BreadcrumbsRoute'.
    Type '{ path: string; breadcrumb: null; }' is not assignable to type 'BreadcrumbsRoute'.
      Types of property 'breadcrumb' are incompatible.
        Type 'null' is not assignable to type 'string | ComponentClass<any, any> | FunctionComponent<any> | ComponentClass<{}, any> | FunctionComponent<{}> | undefined'
```

I can pass `{ path: '/a/b', breadcrumb: undefined }` to silence the error but it will not disable `/a/b` route. Is there any workaround for this or BreadcrumbsRoute type definition needs some updates?

**Comment by icd2k3** [comment]
Thanks @rwojnarowski, I'm going to update how these types are generated soon

**Comment by icd2k3** [comment]
Should be fixed as of `3.3.0`, let me know if there's still a problem!
